_episodes/02-filedir.md

```diff
@@ -67,6 +67,12 @@ which is Nelle's **home directory**:
 > In future examples, we've used Mac output as the default - Linux and Windows
 > output may differ slightly, but should be generally similar.
 {: .callout}
+>  
+> We will also assume that your `pwd` command returns your users home directory. 
+> If `pwd` returns something different you may need to navigate there using `cd ~` 
```

Comment:
Hi, the bare "cd" command also navigates to the home directory, so there's no need for "~"

Reply:
I agree, save any details here for after cd is introduces.